addons/mass_mailing/models/mail_mail.py

```diff
@@ -72,14 +72,16 @@ def send_get_mail_body(self, partner=None):
                     body = body.replace(href, new_href)
 
         # prepend <base> tag for images using absolute urls
-        domain = self.env["ir.config_parameter"].sudo().get_param("web.base.url")
-        base = "<base href='%s'>" % domain
-        body = tools.append_content_to_html(base, body, plaintext=False, container_tag='div')
-        # resolve relative image url to absolute for outlook.com
-        def _sub_relative2absolute(match):
-            return match.group(1) + werkzeug.urls.url_join(domain, match.group(2))
-        body = re.sub('(<img(?=\s)[^>]*\ssrc=")(/[^/][^"]+)', _sub_relative2absolute, body)
-        body = re.sub(r'(<[^>]+\bstyle="[^"]+\burl\(\'?)(/[^/\'][^\'")]+)', _sub_relative2absolute, body)
+        # TMP
+        if body and ('img' in body or 'url' in body):
```

Comment:
@tde-banana-odoo won't this potentially match not only img & url tags but also plain text ?